### ISSUE TITLE
JSON

### ISSUE THREAD

**Issue by ButuzGOL** [heldr/grunt-properties#1]
Can I convert json nested to properties with this plugin ?

Example:

``` json
{
  "a": "b",
  "c": { "d": "e" }
}
```

**Comment by heldr** [comment]
File .properties doesn't support nested attributes.

[http://en.wikipedia.org/wiki/.properties]([http://en.wikipedia.org/wiki/.properties])

**Comment by ZuBB** [comment]
Does this mean that you think next item is not valid?

```
level1.level2 = some string here
```

**Comment by heldr** [comment]
Considering the .properties patterns mentioned in my link above. No, it isn't.

**Comment by ZuBB** [comment]
AF is [OK](http://commons.apache.org/proper/commons-configuration/userguide/howto_properties.html) with nested keys..

Tool that you used to get inspiration (props2js) also supports nested keys

Considering rules in mentioned link above ISO-8859-1 is only allowed codepage, but your tool still works with files that have Unicode chars inside..

# 

Its (almost) zero cost to press a fork button and push changes that I already made, but imho its better to have 1 package that does 1 thing and does it really good.

What  do you think?

**Comment by heldr** [comment]
Thanks for the info. I'm going to check it later. Currently I'm too busy to implement those changes.

BTW it's not my tool, once I published it on github without any licence(I will put MIT), you can do whatever you want with this code. Feel free to pull request or fork. It's your choice ;)

**Comment by ZuBB** [comment]
I will try to cook a pull request.

// It looks like you did not want to accept any other pov except wiki. That's why I go this way

**Comment by heldr** [comment]
No problem,

I reopened this issue to let all users involved tracking your changes :)

**Comment by heldr** [comment]
Fixed on pull request #3
